Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -571,6 +574,8 @@ uint64_t Connection::sendWithAsyncReply(T&& message, C&& completionHandler, uint
     static_assert(!T::isSync, "Async message expected");
 
     if (!isValid()) {
```

Comment:
This is racy, right? 

isValid() changes value on the connections runloop, and this function can be called from any thread, so it could be true here, and false by the time we call addAsyncReplyHandler.